react-hot-button-component/src/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import hotButton from 'hot-button';
+
+let counter = 1;
+
+class CustomButton extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {isClicked: false};
+        this.handleClick = this.handleClick.bind(this);
+    }
+
+    
+
+    handleClick() {
+        this.setState({isClicked: counter++})
+    }
+
+    render() {
+        <hotButton />

```

Comment:
Instead of using this extra component you should just move the contents of that component to here. Also component names need to be capitalized 😄 .